<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>case_info_display</class>
 <widget class="QWidget" name="case_info_display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="textEdit_case_info">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>290</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="treeWidget_multiple_source">
            <property name="minimumSize">
             <size>
              <width>420</width>
              <height>0</height>
             </size>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QTreeView::item{
padding:5px;
}
QTreeWidget::item:has-children
{
       padding:5px;
}
QTreeWidget::item{ 
	 height: 25px; 
 }


QTreeView::item:selected:active{
background-color: transparent;
}

QTreeView::item:selected:!active{
background-color: transparent;

}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
background-color: transparent;

}

QTreeView::branch:!has-children:has-siblings:adjoins-item {
background-color: transparent;

}

QScrollBar{
		width:2px;
}

</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="autoScrollMargin">
             <number>12</number>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="autoExpandDelay">
             <number>-1</number>
            </property>
            <property name="indentation">
             <number>8</number>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>false</bool>
            </property>
            <property name="animated">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="headerHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Source</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>source_name</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_source_info_display">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
